### PR TITLE
Migrated Image from v1 to v2

### PR DESCRIPTION
**Pull request by nikitasheremet**
Moved the Image Component from V1 to V2 and converted file to tsx. Added to StoryBook.

Image component is an Upload Image button.

### CODE REVIEW

File: src/Inputs/Button.tsx [hunk 1]
```diff
@@ -17,17 +17,19 @@ export interface ButtonProps extends MainInterface, ResponsiveInterface {
     loading?: boolean;
     full?: boolean;
     onClick?: React.MouseEventHandler;
+    disabled?: boolean;
```
Comment (fpunny) [inline src/Inputs/Button.tsx]: oof my bad

File: src/Inputs/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    extends MainInterface,
+        ResponsiveInterface,
+        ImplicitPropsInterface {
+    maxWidth?: number | undefined;
+    accept?: string;
+    aspect?: number;
+    onChange?: Function;
+}
+
+const Drop = styled.input`
```
Comment (fpunny) [inline src/Inputs/Image.tsx]: I would move these styled.xxx to the bottom

File: src/Inputs/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+            setImage(URL.createObjectURL(file));
+            modal[1](true);
+        }
+    }, []);
+
+    const onClose = useCallback(() => {
+        URL.revokeObjectURL(image);
+        setCrop({ aspect });
+        img.current = { x: '', y: '', width: '', height: '' };
+        setImage(null);
```
Comment (fpunny) [inline src/Inputs/Image.tsx]: change this to undefined. Null is evil >:c

File: src/Inputs/Image.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+const Drop = styled.input`
+    ${position('absolute', 'auto', 0)}
+    opacity: 0;
+    cursor: pointer;
+    height: 100%;
+    width: 100%;
+`;
+
+const Container = styled.div<ImageProps>`
```
Comment (fpunny) [inline src/Inputs/Image.tsx]: These styles are very outdated, perhaps we should make this a button

File: src/Themes/ThemeTemplate.ts [hunk 5]
```diff
@@ -146,6 +147,7 @@ const Theme: ThemeTemplateInterface = {
         page: 600,
     },
     depth: ['none', '0 1px 3px rgba(0,0,0,0.2)', '0 2px 6px rgba(0,0,0,0.3)'],
+    shadows: ['0 1px 3px rgba(0,0,0,0.2)', '0 3px 8px rgba(0,0,0,0.4)'],
```
Comment (fpunny) [inline src/Themes/ThemeTemplate.ts]: This is the same as depth

File: src/Inputs/Image.tsx (hunk 4 above)
Comment (nikitasheremet) [reply src/Inputs/Image.tsx]: Make the Container component a button instead of a div?

**Pull request merged**